test/controllers/confirmation.controller.spec.unit.ts

```diff
@@ -0,0 +1,76 @@
+import * as request from "supertest";
+import app from "../../src/app";
+import * as pageURLs from "../../src/model/page.urls";
```

Comment:
This file only needs to import the PROMISE_TO_FILE from page.urls

Reply:
Minimsed the imports.